App.js

```diff
@@ -1,26 +1,83 @@
-import React from "react"
-import styled from "styled-components/native"
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components/native';
+import { Text, View, StyleSheet, TouchableOpacity, Share, Image, ImageResizeMode } from 'react-native';
+import Constants from 'expo-constants';
+
+const apiKey = `95622557-9a41-47af-a755-23f39b9b9675`
+const url = `https://api.thedogapi.com/v1/images/search`
+
+
+//reusable fetch function
+
+const fetchDog = () => {
+  return fetch(url, { headers: { Authorization: apiKey} })
+    .then((res) => res.json())
+    .then((json) => {
+      const dogsWithBreedInfo = json.filter((dog) => dog.breeds.length > 0)
+      if (dogsWithBreedInfo.length > 0) {
+        return dogsWithBreedInfo
+      } else {
+        return fetchDog()
+      }
+    })
+}
 
 const App = () => {
-  return (
-    <Container>
-      <Title>This is your cool app!</Title>
-      <Title>Go to App.js and start coding</Title>
-      <Title>💅💅💅</Title>
+  const [photos, setPhotos] = useState([])
+
+  const fetchDogPhotos = () => {
+    fetchDog()
+      .then((json) => {
+        setPhotos(json)
+        console.log(json)
+      })
+  }
+
+  useEffect(() => {
+    fetchDogPhotos()
+  }, [])
+
+
+    return(
+     <Container>
+     <TouchableOpacity onPress={fetchDogPhotos}> 
+      <View>
+         {photos.map((photo) => (
+           <View key={photo.id}>
+           {/*<Text>{photo.breeds[0].name}</Text> //dont work for all apis*/}
```

Comment:
This code works, the problem why you can't see it is because the background is black and the default text color is also black.
Change the textcolor or the background color and you'll see it!